viz_scripts/energy_calculations.ipynb

```diff
@@ -146,24 +117,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "## Mapping new labels with dictionaries\n",
+    "# Mapping new labels with dictionaries\n",
     "expanded_ct['Trip_purpose']= expanded_ct['purpose_confirm'].map(dic_pur)\n",
     "expanded_ct['Mode_confirm']= expanded_ct['mode_confirm'].map(dic_re)\n",
     "expanded_ct['Replaced_mode']= expanded_ct['replaced_mode'].map(dic_re)\n",
     "\n",
-    "#Mapping fuel\n",
+    "# Mapping fuel\n",
     "expanded_ct['Mode_confirm_fuel']= expanded_ct['Mode_confirm'].map(dic_fuel)\n",
-    "expanded_ct['Replaced_mode_fuel']= expanded_ct['Replaced_mode'].map(dic_fuel)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "micro-wound",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "scaffolding.unit_conversions(expanded_ct)"
+    "expanded_ct['Replaced_mode_fuel']= expanded_ct['Replaced_mode'].map(dic_fuel)\n",
+    "\n",
+    "# Change meters to miles\n",
+    "scaffolding.unit_conversions(expanded_ct)\n",
+    "\n",
+    "# Calculate energy impact\n",
+    "expanded_ct = scaffolding.energy_intensity(expanded_ct, df_ei, 'distance_miles', 'Replaced_mode', 'Mode_confirm')\n",
+    "expanded_ct = scaffolding.energy_impact_kWH(expanded_ct, 'distance_miles', 'Replaced_mode', 'Mode_confirm')\n",
+    "expanded_ct = scaffolding.CO2_impact_lb(expanded_ct, 'distance_miles', 'Replaced_mode', 'Mode_confirm')"
```

Comment:
This order is less computationally efficient, but since some notebooks use combinations of filtered/non filtered trips by mode of interest moving the filter earlier could be some repeated code in various places to make sure data frames that need energy info always have it.